models/articles.model.js

```diff
@@ -19,4 +19,16 @@ exports.selectArticleById = (article_id) => {
         if(!results.rows.length) return Promise.reject({"status": 404, msg: "article not found!"})
         return results.rows[0]
     })
+}
+exports.updateArticle = (article_id, obj) => {
+    return this.selectArticleById(article_id).then(() => {
```

Comment:
This is a bit clunky. You would expect the controller to invoke models rather than a model invoking another model. Also think about how you could increase the votes from within a single SQL statement rather than having to make multiple successive queries.